src/main/xml-parsers.js

```diff
@@ -96,7 +96,8 @@ export function parseListMultipart(xml) {
 
   if (xmlobj.CommonPrefixes) {
     toArray(xmlobj.CommonPrefixes).forEach(prefix => {
-      result.prefixes.push({prefix: toArray(prefix)[0]})
+      const prefixVal = sanitizeObjectKey(toArray(prefix.Prefix)[0])
+      result.prefixes.push({prefix: prefixVal})
```

Comment:
We can get rid of the `const` definition and make the code shorter:
```suggestion
      result.prefixes.push({prefix: sanitizeObjectKey(toArray(prefix.Prefix)[0])})
```
But if you think the existing code is more understandable and has a better readability, then it is fine.

Reply:
@ebozduman Thank you for the suggestion. I have updated.